### ISSUE TITLE
Wrong values when core isolation (hvci) is on

### ISSUE THREAD

**Issue by treepoint** [treepoint/AveTemp#20]
There is the wrong values for frequency and load when core isolation (hvci) is on. Need to be tested with LibreHardwareMonitorLib standalone and if bug will reproduce then must to check newer version of the library and, if it will not help, report to them about bug.

![20230122_213207](https://user-images.githubusercontent.com/25937222/213974078-8fa9c911-8e66-4895-9ad8-6632a5db4a5b.jpg)

**Comment by treepoint** [comment]
Can be connected with:

- https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/566
- https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/645

Only core isolation is not enough to reproduce, cause, as it seems, all this time i was using AveTemp with core isolation on and it was okay.

**Comment by treepoint** [comment]
And another detail: it's working perfectly fine with win 11 though.

**Comment by treepoint** [comment]
Update LibreHardwareMonitorLib to latest nighty build from 24 september 2023 not help.

**Comment by DjordjeMandic** [comment]
Any progress on this? Voltages are also off

![image](https://user-images.githubusercontent.com/6750655/224183614-a758551c-ac16-4742-87e9-713c5af1fc0c.png)

**Comment by treepoint** [comment]
> Any progress on this? Voltages are also off

Not at all, sorry. I can't reproduce that bug on any machine that i have, so only option for me is to wait fixes from LHMLib for now. 

I can build test version with latest nightly LHMLib, if so, could i ask you to test it and tell me results?

**Comment by DjordjeMandic** [comment]
I'm up for it. If you wish create a fork and link it, i will pull it and build it. I'm on r7 1700, x370 with hyper-v on and svm enabled. I can try to debug it also if i find some free time.

**Comment by treepoint** [comment]
Sorry for delay.

I don't need to fork anything to build nightly LHMLib with AveTemp. I had downloaded the latest build [here](https://nightly.link/LibreHardwareMonitor/LibreHardwareMonitor/workflows/master/master/LibreHardwareMonitor-net472.zip) and build AveTemp with new DLL.

Please, check [this version](https://drive.google.com/file/d/15gu5FZwVL9bKsbiBxrm4UjYBL9LA8xDV/view?usp=share_link), does it work better with you platform?

**Comment by treepoint** [comment]
> I tought i was in LHM repo xd lul

I supposed that you thought so, but i don't have many testers and i decide to try fortune and ask you to test :D

It's sad that it didn't work, but anyway thank you for testing and feedback)
